src/common/routes.js

```diff
@@ -14,5 +15,6 @@ export default (
     <Route path="/:owner/:name/builds" component={Builds}/>
     <Route path="/:owner/:name/builds/:buildId" component={BuildDetails}/>
     <Route path="/login/failed" component={LoginFailed}/>
+    <Route path="/dashboard/select-repositories" component={SelectRepositories}/>
```

Comment:
I think the dashboard definitely needs a route of its own.

- It allows deep-linking of the dashboard and the "select-repositories" page
- It allows users to view dashboards when logged out
- We can get the benefits of hooking into the browser history API